addons/addon-base-raas-ui/packages/base-raas-ui/src/models/aws-accounts/AwsAccount.js

```diff
@@ -95,13 +96,14 @@ const AwsAccount = types
       self.description = rawAwsAccounts.description || self.description;
       self.accountId = rawAwsAccounts.accountId || rawAwsAccounts.accountId;
       self.externalId = rawAwsAccounts.externalId || self.externalId;
-      self.permissionStatus = rawAwsAccounts.permissionStatus || self.permissionStatus || 'PENDING';
+      self.permissionStatus = rawAwsAccounts.permissionStatus || self.permissionStatus || 'NEEDSONBOARD';
```

Comment:
shouldn't be a big deal to rename them.

PENDING accounts are handled separately, so the permission checker only looks at accounts not in PENDING state (auto-skipping pending accounts) and then there's a separate functionality that looks at accounts in PENDING and pulls them out of it. Accounts go into PENDING state when their CFN stack is in a pending state. PENDING is definitely required because it indicates to the user that there's changes happening, otherwise there can be issues where the user makes changes but it doesn't look like anything has happened

Reply:
as for this line specifically, it's just the default value if something goes wrong and we don't have a permission status...I think UNKNOWN is probably the right choice to have there rather than PENDING or NEEDS_ONBOARD

Reply:
Renamed that everywhere it occurs, and I changed this line to UNKNOWN as the default 'missing' value

Reply:
yep, UNKNOWN sounds good.